bundler/src/bundle.rs

```diff
@@ -110,20 +115,35 @@ where
             sessions,
             proposals,
             beamlines,
+            static_data,
         }
     }
 
     /// Fetches [`Subjects`] from ISPyB and constructs a [`Bundle`]
     #[instrument(name = "fetch_bundle")]
-    pub async fn fetch(metadata: Metadata, ispyb_pool: &MySqlPool) -> Result<Self, sqlx::Error> {
+    pub async fn fetch(
+        metadata: Metadata,
+        static_data_directory: Option<&Path>,
+        ispyb_pool: &MySqlPool,
+    ) -> Result<Self, sqlx::Error> {
         let (subjects, sessions, proposals, beamlines) = try_join!(
             Subjects::fetch(ispyb_pool),
             Sessions::fetch(ispyb_pool),
             Proposals::fetch(ispyb_pool),
             Beamlines::fetch(ispyb_pool),
         )?;
+        // Ignore all static data if any of the reading fails
+        let static_data = match static_data_directory {
```

Comment:
Would rather fail loudly here. Maybe even crash

Reply:
Now returns the error here as well as in the static_data function